xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/BaseProperty.java

```diff
@@ -261,7 +265,8 @@ public void merge(ElementInterface previousElement, ElementInterface newElement,
      * @param mergeResult merge report
      * @since 3.2M1
```

Comment:
The since is not accurate anymore.

Reply:
> There was really no way to keep the old one, just not used ?

No I could probably keep the old one and make it deprecated. Wasn't sure it worth the effort as I don't see it much used in XS so I don't expect it to have been used. But if you feel we should keep it I can.

Reply:
Better keep APIs, just in case, you never know what one might be doing in some extension and the promise is supposed to be that it's not going to be broken.

Reply:
Done with the new changes, but since I tried not using a deprecated class in argument, the changes became much more complex... sorry for that...